doc/include/api_stubs.inc

```diff
@@ -1,122 +1,96 @@
+mapping macros, etc., and may include package-specific headers.  See
+[cmd {critcl::api header}], below.  An [term {#include}] directive is added at
+the beginning of the generated code for [vset critcl_script] and at the
+beginning of each of its companion [file .c] files.
+
+[para] [var name]/[var name]StubLib.h contains the stubs table variable
+definition and the function to initialize it.  An [term {#include}] directive
+for it is added to the initialization code for the [vset critcl_script] ,
+along with a call to the initializer function.
+
+[para] If [file [var name]/[var name].decls] accompanies
+[var name]/[var name]Decls.h, it should contain the external representation of
+the stubs table used to generate the headers. The file is read and the internal
+representation of the stubs table returned for use by the importing package.
+Otherwise, the empy string is returned.
 
 [para] One possible use would be the automatic generation of C code
 calling on the functions listed in the imported API.
 
-[para] When generating a TEA wrapper the names of the imported APIs
+[para] When generating a TEA package the names of the imported APIs
 are used to declare [syscmd configure] options with which the user can
-declare a non-standard location for the headers of the API. Any API
-[var FOO] is translated a single configure option
-[option --with-[var FOO]-include].
+declare a non-standard directory for the headers of the API. Any API
+[var name] is translated into a single configure option
+[option --with-[var name]-include].
 
 [list_end]
 
 
-[para] Second, declaration and export of a stubs table, i.e. API, for
-the current package, [var foo]:
+[para] Declaration and export of a stubs table, i.e. API, for
+the [vset critcl_script]:
 
 [list_begin definitions]
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::api] [method function] [arg resulttype] [arg name] [arg arguments]]
 
-This method declares that the function [arg name] is in the public API
-of the package, and its signature (type of the result, number, names
-and types of its arguments).
-
-Using this method automatically causes critcl to generate both the
-code for a stubs table in the package, the headers needed by packages
-using this API, and a [file .decls] file containing the stubs table
-implied by the exports, usable by [cmd {critcl::api import}].
-
-[para] [arg arguments] is a list of C types and associated argument
-names. Like a dictionary, except that keys (argument names) and values
-(argument types) are swapped. The [arg resulttype] is a C type as well.
+Adds to the public API of the [vset critcl_script] the signature the
+for the function named name [arg name] having the signature specified by
+[arg arguments] and [arg resulttype].  Code is generated for a [file .decls]
+file, the corresponding public headers, and a stubs table usable by
+[cmd {critcl::api import}].  [para] [arg arguments] is a multidict where each
```

Comment:
Move `[para]` to a new line

Reply:
Done with commit 29380521d0873ea1dcc5f8cdff6742b20764473f (Branch `documentation` in my repo, not seen by the PR).